packages/app/components/childListItem.component.tsx

```diff
@@ -232,6 +232,7 @@ export const ChildListItem = ({ child, color }: ChildListItemProps) => {
       )}
       <View style={styles.itemFooterAbsence}>
         <Button
+          accessible={true}
```

Comment:
```suggestion
          accessible
```

`true` can be implicitly passed by just adding the prop